project/profiles/capacity4more/behat/features/group_dashboard.feature

```diff
@@ -135,3 +135,10 @@ Feature: Group dashboard
     | galileo     |
     | badhairday  |
     | president   |
+
+  @api
+  Scenario: Highlighted document content title should contain a link.
+    Given I am logged in as user "badhairday"
+    When  I visit the dashboard of group "Nobel Prize"
+    Then  I should see the text "Nobel Prize"
+    And I should see the link "Nobel Prize sheet"
```

Comment:
> After migration, "Nobel Prize sheet" link appears on group dashboard only once, at 'Highlighted' block. Why do we need to access the link? Isn't it enough to make sure we see it's a link.

Not true , it is also appear on the activity stream.

> There's no 'highlight box class'. Adding it to all content types will take time.
Creating 'I visit the highlighted link ..' function will take time.

Again, not true, `.view-c4m-overview-og-highlights` is the class you want to use.

![selection_382](https://cloud.githubusercontent.com/assets/5985683/23759440/81f69ea6-04f5-11e7-95fe-db11f5bfe358.png)

____
> I don't think this test worth the time that need to be spent.

Sorry for not agreeing with you, but I think it is, otherwise I would not ask you for that.